docs/index.html

```diff
@@ -538,6 +542,184 @@ <h2 class="heading-medium">Support</h2>
+
+    <div class="grid-row">
+
+        <div class="column-two-thirds">
+
+            <h2 class="heading-medium">Introduction</h2>
+
+            <p>The HGV/PSV Operators Annual Test History API provides a mechanism for authorised third party
+                organisations to access Annual Test History information for HGV’s, PSV,s and Trailers through an API. </p>
+
+            <h2 class="heading-medium">Apply for API access</h2>
+
+            <p><a href="https://www.smartsurvey.co.uk/s/MOT_History_TradeAPI_Access_and_Support/" target="_blank">
+                Complete an application form </a> giving details of your organisation and your intended use of the API.
+                If your application is approved you will be given an API key that will let you access the MOT history data.
+            </p>
+
+            <h2 class="heading-medium">Getting started</h2>
+
+            <p>All Annual Test History API interfaces are implemented as restful APIs and accessed over https. In order
+                to access the API you will require an API Key that uniquely identifies the source of the request. API
+                Keys should be treated as sensitive information as throttling and quotas are managed at an API Key level.</p>
+
+            <p>To access API you will need an API key that uniquely identifies the source of the request. DVSA will give
+                you an API key if it approves your application.</p>
+
+            <p>Each request must have two mandatory fields in the header to enable you to access the data:</p>
+
+            <ul class="list-bullet u-space-b30">
+                <li>Accept: application/json</li>
+                <li>x-api-key: &lt;your api key&gt;</li>
+            </ul>
+
+            <p>The first field ensures that the response type is in json format and the second field provides your API
+                key to identify the source of the request. You will not be able to access the API without providing a
+                valid API Key. </p>
+
+            <h2 class="heading-medium">Technical resources</h2>
+
+            <p>The trade API can currently be accessed using the following root URL: <a
+                    href="https://beta.check-mot.service.gov.uk/" target="_blank">https://beta.check-mot.service.gov.uk/</a></p>
+
+            <ol class="list-number">
+                <li>
+                    <p class="responsive-code">/trade/vehicles/annual-tests?registrations={registration}</p>
+                </li>
+            </ol>
+
+            <h2 class="heading-medium">Examples</h2>
+
+            <p>The following examples use the curl command to illustrate how to use the three different methods:</p>
+
+            <ul class="list-number">
+
+                <li>
+                    <p class="responsive-code">1.	curl -H "Accept: application/json" -H "x-api-key: &lt;your_api_key&gt;" \https://beta.check-mot.service.gov.uk/trade/vehicles/annual-tests\?registrations=MM18OOH</p>
```

Comment:
Does this `1.` need to be here? The number is duplicated

![Screenshot 2020-10-07 at 10 03 09](https://user-images.githubusercontent.com/14791708/95310536-6cdc1500-0884-11eb-85cb-7b0cf0ab3577.png)

Reply:
I have removed this